app/requirements-links.txt

```diff
@@ -1 +1 @@
-git+https://github.com/eclipse-velocitas/vehicle-app-python-sdk.git@v0.9.0
+git+https://github.com/eclipse-velocitas/vehicle-app-python-sdk.git@fix/dont_use_index-based_access
```

Comment:
It is not for merge. Please update it after SDK will be release

Reply:
Done